### ISSUE TITLE
Installing Issues - Objects not created

### ISSUE THREAD

**Issue by andreash1027** [larshp/ABAP-Swagger#38]
I need this repo primarily to use abapgitserver.

I have issues when trying to clone the Swagger Repo via AbapGit.
The objects are not created in the system.

![image](https://user-images.githubusercontent.com/28306535/42364123-6222bb14-80fa-11e8-80ba-615a73e0cbd0.png)

The download completes without a problem, also there are no exceptions thrown in the debugger.

![image](https://user-images.githubusercontent.com/28306535/42364066-354faa70-80fa-11e8-96ed-d13c5555df0c.png)

Other repos, like ABAP2XLSX, were successfully installed in our system.

Is there any setting or naming convention to follow?

Following solutions were tested:
 - Naming the package with a Z ("ZABAPSWAGGER")
 - Creating a local package ("$ABAPSWAGGER")
 - Creating the package in Namespace
 - Cloning the repo when logged in in English language

**Comment by larshp** [comment]
try copy pasting the classes manually and see what happens

**Comment by andreash1027** [comment]
I was able to copy the classes and transformation and to activate them.

Now I had the same issue for the AbapGitServer.

I tried the package name "ZABAPGITSERVER" as well as "$ABAPGITSERVER", both did not work.

But after clicking on "reset local" the program asked me to overwrite one object (the package) and now tried to activate all of the repo's objects.

So, the workaround seems to reset the local after importing the repo.
